tests/test_android.py

```diff
@@ -92,8 +93,8 @@ def test_to_native_coordinates(
         screenshot_x, screenshot_y, coordinate_system,
         _Dimensions(*device_resolution))
     print(native_x, native_y)
-    assert isclose(native_x, expected_coordinates[0], absolute_tolerance=1)
-    assert isclose(native_y, expected_coordinates[1], absolute_tolerance=1)
+    assert isclose(native_x, expected_coordinates[0], atol=1)
+    assert isclose(native_y, expected_coordinates[1], atol=1)
```

Comment:
`pytest.Approx` is even nicer.

Reply:
Yeah, we'd have to upgrade pytest in .travis.yml.